migrations/migrations.go

```diff
@@ -114,3 +123,38 @@ func MigrationOptions() *gormigrate.Options {
 	options.ValidateUnknownMigrations = true
 	return options
 }
+
+func createTableState(tx *gorm.DB) error {
+	if err := tx.CreateTable(&models.State{}).Error; err != nil {
```

Comment:
"it's a good practice to copy the struct inside the function, so side effects are prevented if the original struct changes during the time" (c) https://github.com/go-gormigrate/gormigrate

Reply:
Done